components/carousel/module/CarouselMedia.vue

```diff
@@ -5,23 +5,17 @@
     <nuxt-link
       :to="urlOf({ id: item.id, url, chain: item.chain })"
       rel="nofollow">
-      <PreviewMediaResolver
-        v-if="item.animationUrl"
-        :src="item.animationUrl"
-        :poster="imageSrc || ''"
-        :metadata="item.metadata" />
-      <BasicImage
-        v-else
-        :src="imageSrc"
-        :alt="item.name"
-        custom-class="carousel__image-wrapper" />
```

Comment:
How is 3D handled then?

Reply:
`<MediaItem />` component
https://github.com/kodadot/nft-gallery/blob/6d4cf92216089b2798410690d2d0e7ba3c2fd3b9/libs/ui/src/components/MediaItem/MediaItem.vue#L91-L94